frontend/src/bundles/talent-onboarding/talent-onboarding-api.ts

```diff
@@ -36,6 +36,24 @@ class TalentOnBoardingApi extends HttpApiBase {
         return response.json<UserDetailsGeneralCustom>();
     }
 
+    public async updatePublishedData(
+        payload: Partial<UserDetailsGeneralCustom>,
+    ): Promise<UserDetailsGeneralCustom> {
+        const { userId = '' } = payload;
```

Comment:
Why we do this = '' ?

Reply:
I think userId can't be null in this api method, because we need to handle it previously

Reply:
Agree. Fixed